policy/pipeline/task_bundle.rego

```diff
@@ -79,7 +79,8 @@ warn[result] {
 #   are acceptable given the tracked effective_on date. See the list of acceptable
 #   task bundles at xref:acceptable_bundles.adoc#_task_bundles[Acceptable Bundles] or look at
 #   link:https://github.com/hacbs-contract/ec-policies/blob/main/data/acceptable_tekton_bundles.yml[data/acceptable_tekton_bundles.yml]
-#   in this git repository.
+#   in this git repository. The meaning of an acceptable bundle is explained in
+#   xref:acceptable_bundles.adoc#_task_bundles[Acceptable Bundles]
```

Comment:
@lcarva Does is it also necessary to update the meaning of acceptable bundle for line 65 of ` Task bundle is out of date`?

Reply:
https://github.com/hacbs-contract/ec-policies/pull/167